base/sparse/sparsevector.jl

```diff
@@ -1118,13 +1109,13 @@ end
 
 ### Binary arithmetics: +, -, *
 
-for (vop, fun, mode) in [(:_vadd, :AddFun, 1),
-                         (:_vsub, :SubFun, 1),
-                         (:_vmul, :MulFun, 0)]
+for (vop, fun, mode) in [(:_vadd, :+, 1),
```

Comment:
Carry on, I thought my comment from above applied somewhere else too, and after you updated the commits I got confused.